main.go

```diff
@@ -10,40 +11,40 @@ import (
 
 func main() {
 	// Needs as argument a path to an Antora project
-	if len(os.Args) < 2 {
-		fmt.Println("Please provide a path to an Antora project")
-		os.Exit(1)
-	}
-	path := os.Args[1]
+	var antoraPath string
+	flag.StringVar(&antoraPath, "antoraPath", "/docs", "Path to Antora document sources")
 
-	// Remove the trailing slash
-	if strings.HasSuffix(path, "/") {
-		path = strings.TrimSuffix(path, "/")
-	}
+	// Module to check. If none passed, it is assumed to be ROOT
+	var module string
+	flag.StringVar(&module, "module", "ROOT", "Module to analyze")
+
+	// File to parse and analyze; if none is passed, it is assumed to
+	// be a standard Antora navigation file at the ROOT module.
+	var filename string
+	flag.StringVar(&filename, "filename", "/modules/ROOT/nav.adoc", "File to analyze")
+
+	flag.Parse()
 
 	// Main block
-	errors := check(path)
+	errors := check(antoraPath, module, filename)
 
 	// If there are errors, print the list of orphan files
 	if len(errors) > 0 {
 		for _, file := range errors {
-			fmt.Println(fmt.Sprintf("File '%s' not in 'nav.adoc'", file))
+			fmt.Fprintf(os.Stdout, "File '%s' not in '%s'\n", file, filename)
```

Comment:
```suggestion
			fmt.Fprintf(os.Stderr, "File '%s' not in '%s'\n", file, filename)
```

Oh no, I actually wanted this to be written to STDERR. 🤦🏼